src/drivers/boards/px4fmu-v4/px4fmu_spi.c

```diff
@@ -103,7 +103,16 @@ __EXPORT void stm32_spi1select(FAR struct spi_dev_s *dev, enum spi_dev_e devid,
 		px4_arch_gpiowrite(GPIO_SPI_CS_MPU9250, 1);
 		px4_arch_gpiowrite(GPIO_SPI_CS_HMC5983, 1);
 		px4_arch_gpiowrite(GPIO_SPI_CS_MS5611, 1);
-		px4_arch_gpiowrite(GPIO_SPI_CS_ICM_20608_G, !selected);
+		px4_arch_gpiowrite(GPIO_SPI_CS_ICM_2060X_G, !selected);
+		break;
+
+	case PX4_SPIDEV_ICM_20602:
```

Comment:
I am assuming that there is only one U location and it can be populated with either a 20602 or a 20608 - so that no FMUv4 board will ever have the 2 IC at the same time.

In that case: Delete this block and insert line 109 at 99-100.